src/components/VercelCallout.js

```diff
@@ -0,0 +1,26 @@
+import { Box, Link } from "@chakra-ui/react"
+
+export function VercelCallout() {
+  return (
+    <Box
+      mt="6"
+      fontSize="sm"
+      fontWeight="semibold"
+      display="inline-block"
+      bg="black"
+      color="white"
+      px="4"
+      py="2"
+      rounded="lg"
+    >
+      {" "}
+      <Link href="https://vercel.com?utm_source=xarray&utm_campaign=oss">
```

Comment:
The docs mention 

> All links back to Vercel should use a UTM tag with the format: `?utm_source=[team-name]&utm_campaign=oss`.

I presume `[team-name]` in `utm_source=[team-name]` is a placeholder. This is why I replaced it with `xarray`. 

@jhamman, let me know if this is correct.

Reply:
100% correct!